Jucer2CMake/src/reprojucer.hpp

```diff
@@ -2068,39 +2069,6 @@ inline void writeReprojucerCMakeLists(const Arguments& args,
         const auto targetProjectDir =
           jucerFileDir.getChildFile(exporter.getStringAttribute("targetFolder"));
 
-        const auto convertSearchPaths =
-          [&isAbsolutePath, &jucerFileDir,
-           &targetProjectDir](const juce::String& searchPaths) -> juce::StringArray {
-          if (searchPaths.isEmpty())
-          {
-            return {};
-          }
-
-          juce::StringArray absOrRelToJucerFileDirPaths;
-
-          for (const auto& path : juce::StringArray::fromTokens(searchPaths, ";\r\n", {}))
-          {
-            if (path.isEmpty())
-            {
-              continue;
-            }
-
-            const auto unquotedPath = path.unquoted();
```

Comment:
This line is important and should not be lost. Please preserve it in the new function.